cli/cli/cli.go

```diff
@@ -452,6 +457,48 @@ func (c *CLI) logLevel() string {
 	return c.config.GetString("rexray.logLevel")
 }
 
+func (c *CLI) isConnSecured() {
```

Comment:
Hi @vladimirvivien,

I just searched for this function against this PR's diff and didn't see it pop up anywhere else. Is this for debugging only? It does not appear to be used.